scripts/zones/Vunkerl_Inlet_[S]/mobs/Pallas.lua

```diff
@@ -2,9 +2,111 @@
 -- Area: Vunkerl Inlet [S]
 --   NM: Pallas
 -----------------------------------
+mixins = { require('scripts/mixins/job_special') }
+local ID = zones[xi.zone.VUNKERL_INLET_S]
+-----------------------------------
 local entity = {}
 
+local function callTigerPet(mob, target)
+    local act         = mob:getCurrentAction()
+    local petsSpawned = mob:getLocalVar('petsSpawned')
+    local isBusy      = false
+
+    if
+        act == xi.act.MOBABILITY_START or
+        act == xi.act.MOBABILITY_USING or
+        act == xi.act.MOBABILITY_FINISH
+    then
+        isBusy = true
+    end
+
+    if
+        mob:actionQueueEmpty() and
+        not isBusy
+    then
+        if petsSpawned ~= 1 then
+            -- Begin casting animation
+            mob:timer(1000, function(mobArg)
```

Comment:
I have not dug through packets to check. I started to and honestly bailed out with a migraine. but I really doubt this should happen via timer and forced animations instead of some actual action by the mob like other mobs with pets are doing. The problem with reusing what other mobs do is they only get 1 pet each, so I know we can't do that exactly, but I think that functionality is going to need improved by someone with the know-how for that so that you can use it.

Reply:
> pet summoning is really jank sometimes and it can literally just be an injected action packet with no "real" action behind it and the pets just come out

even if that's what we're doing timer-ing it is not a long term solution we should retain/repeat for these few multi-pet mobs, so at best this is a "todo" so I had him open an issue about it. And double uh-uh to nesting or multiple timer calls to do ..anything really.